### DISCUSSION TITLE
[@types/aws-lambda] vpc lattice trigger

### DISCUSSION THREAD

**Discussion by dave-graham** [DefinitelyTyped/DefinitelyTyped#65699] [Issues with a @types package]
Hi,

We currently use @types/aws-lambda to type requests and responses to our lambdas. I'd like to add vpc lattice ( https://docs.aws.amazon.com/vpc-lattice/latest/ug/what-is-vpc-lattice.html ) in as a trigger, but I can't see it in @types/aws-lambda. Am I missing it, or is it something that needs adding in?

[a comment by an automated account is not shown]

**Comment by darbio** [comment]
Hi @dave-graham. I don't think we have it in here yet. Feel free to submit a PR with it in and we can merge it into the types.
